src/shared/components/Carousel/Carousel.module.scss

```
@@ -31,12 +35,13 @@
   flex: 0 0 100%;
   background: f.color(white);
   @include m.flex-box(flex-start, stretch, column);
-  gap: 24px;
-  padding: 32px 32px 24px 32px;
+  gap: 12px;
```

Stylelint (17.3.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)